package.json

```diff
@@ -14,6 +14,7 @@
     "@testing-library/user-event": "^7.1.2",
     "@types/jest": "^24.0.0",
     "@types/jwt-decode": "^2.2.1",
+    "@types/lodash": "^4.14.182",
```

Comment:
What is this new package for?

Reply:
[debounce](https://lodash.com/docs/4.17.15#debounce)

Reply:
@Serhii-Buniak , you also might look in this debounce function, might come in handy